ndb_api/db_helpers.go

```diff
@@ -109,22 +83,15 @@ func GenerateProvisioningRequest(ctx context.Context, ndb_client *ndb_client.NDB
 				Name:  "dbserver_description",
 				Value: "dbserver for " + database.GetDBInstanceName(),
 			},
-			{
-				Name:  "database_names",
-				Value: database_names,
-			},
-			{
-				Name:  "db_password",
-				Value: dbPassword,
-			},
 			{
 				Name:  "database_size",
 				Value: strconv.Itoa(database.GetDBInstanceSize()),
 			},
 		},
 	}
-	// Setting action arguments based on database type
-	requestBody.ActionArguments = append(requestBody.ActionArguments, database.GetDBInstanceActionArguments()...)
+	// Appending request body based on database type
+	reqAppender, err := GetDbProvRequestAppender(database.GetDBInstanceType())
```

Comment:
Can we call it requestAppender instead of reqAppender since the variable in the next line says `requestBody`. Just trying to be consistent in the same function.